playground/frontend/playground_components/test/src/common/examples.dart

```diff
@@ -50,6 +50,7 @@ const exampleWithoutSourceMock = ExampleBase(
   description: 'description',
   path: 'SDK_PYTHON/Category/Name',
   complexity: Complexity.basic,
+  datasets: [],
```

Comment:
`const []`? Or delete?

Reply:
all variable is const, so const here is redundant

Reply:
done